playbooks/kind-integration-test-arm64/run.yaml

```diff
@@ -16,15 +17,14 @@
           go get sigs.k8s.io/kind
           go get k8s.io/kubeadm
           go get k8s.io/kubernetes
-          go get k8s.io/test-infra
+          go get k8s.io/test-infra/kubetest
         executable: /bin/bash
       environment: '{{ global_env }}'
     - name: build KIND against k/k master using bazel
       shell:
         cmd: |
           cd $GOPATH/src/sigs.k8s.io/kind && go install .
           kind build base-image --image kindest/base:latest --source $GOPATH/src/sigs.k8s.io/kind/images/base --loglevel debug
-          kind build node-image --type bazel --image kindest/node:latest --base-image kindest/base:latest --kube-root=$GOPATH/src/k8s.io/kubernetes/ --loglevel debug
         executable: /bin/bash
```

Comment:
The image will be build again in kubetest command, so let's not build it here since it takes about 20 mins and we can speed up by remove this